resources/public/api/conf/1.0/application.raml

```diff
@@ -872,6 +863,13 @@ types:
                                     "code" : "SUPERSEDED_FUND_RELEASE_MISMATCH_ERROR",
                                     "message" : "originalFundReleaseId and the originalEventDate do not match the information in the original request"
                                   }
+                              SupersededFundReleaseDataProvided:
+                                description: You must only give an originalFundReleaseId and originalEventDate when the fund release reason is supersede
```

Comment:
Need to take this out. It's out of scope of the current story, and it's factually inaccurate.